inc/levelagreement.class.php

```diff
@@ -267,7 +267,9 @@ function showForTicket(Ticket $ticket, $type, $tt, $canupdate) {
             echo Dropdown::getDropdownName(static::getTable(),
                                            $ticket->fields[$laField])."&nbsp;";
             echo Html::hidden($laField, ['value' => $ticket->fields[$laField]]);
-            $comment = isset($this->fields['comment']) ? $this->fields['comment'] : '';
+            $obj = clone $this;
```

Comment:
Cloning can have unexpected behaviour, and for this usage, a usage of `new` may be more appropriate, as we do not care of current state of `$this`.
```suggestion
            $obj = new static();
```